src/example.config.ts

```diff
@@ -20,6 +21,8 @@ export const Config: ConfigType = {
         enabled: false,
     },
 
+    shards: 2,
```

Comment:
This should default set to `'auto'` so it always uses the recommended shard count from [`/gateway/bot`](https://discord.com/developers/docs/topics/gateway#get-gateway-bot)